### PR TITLE
enable cluster settings integration target to run on eco vcenter

### PR DESCRIPTION
**Pull request by shellymiron**
See [jira](https://issues.redhat.com/browse/ACA-1636) ticket for this effort

### CODE REVIEW

File: tests/integration/targets/vmware_ops_cluster_settings_test/run.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      ansible.builtin.include_vars:
+        file: vars.yml
+      tags: integration-ci
+
+    - name: Vcsim
+      ansible.builtin.import_role:
+        name: prepare_soap
+      tags: integration-ci
+
+    - name: Import provision VM role
```
Comment (elsapassaro) [inline tests/integration/targets/vmware_ops_cluster_settings_test/run.yml:22]: `- name: Import cluster_settings role`

Comment (shellymiron) [reply tests/integration/targets/vmware_ops_cluster_settings_test/run.yml]: opsss, fixed

File: tests/integration/targets/vmware_ops_cluster_settings_test/runme.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+ANSIBLE_TAGS=$(awk '/ansible_tags/ {print $2}' ../../integration_config.yml)
+
+# Check if the ANSIBLE_TAGS variable is set
+if [[ -n "$ANSIBLE_TAGS" ]]; then
+  echo "ANSIBLE_TAGS is set to: $ANSIBLE_TAGS"
+  exec ansible-playbook run.yml --tags "$ANSIBLE_TAGS"
+else
+  echo "ANSIBLE_TAGS is not set for Eco vCenter. Running on simulator."
+  exec ansible-playbook run.yml --tags integration-ci
+fi
```
Comment (machacekondra) [inline tests/integration/targets/vmware_ops_cluster_settings_test/runme.sh:14]: I see this is reused everywhere, maybe worth to create symlink?

**Review by machacekondra — APPROVED** [review]

**Review by elsapassaro — APPROVED** [review]

File: tests/integration/targets/vmware_ops_cluster_settings_test/runme.sh (hunk 2 above)
Comment (shellymiron) [reply tests/integration/targets/vmware_ops_cluster_settings_test/runme.sh:14]: yeah I agree, it is in my todo tasks :D
let's merge this one and I'll open a new PR for this modification?

**Review by bardielle — APPROVED** [review]

**Pull request merged**